### PR TITLE
py3-cmd: allow sending events with more options 

### PR DESCRIPTION
**Pull request by lasers**
Making it an official PR for @ultrabug to test it.

This is an alternative to https://github.com/ultrabug/py3status/pull/1358 to resolve py3-cmd issue https://github.com/ultrabug/py3status/issues/1349. Additionally, this should close insufficient documentation issue too https://github.com/ultrabug/py3status/issues/1245.

Changelog:
* I added documentation everywhere. Closes an issue (imo).
* In timer, I rename `mins` to `minutes` for cleaner example in `py3-cmd` docs.
* Unrelated. I rename `output` to `verbose` making it less vague.
* I understand why we didn't have aliases or button number in first place. If we work with `argparse`, it tends to sort stuffs and give us args to use. Some things are done after processing arguments. To keep it clean/simple, we would want to try having `argparse` deal with all arguments.
* All old arguments should still work. The only difference is that I stopped documenting aliases and button number and start documenting things the way `argparse` would want it documented (ie explicitly). Aliases are usually done via loop and the button number is really a module name in argparse eyes... which we convert it later to a button.
* EDIT: If there are an error, I'll print `help` right away to reduce steps.

~~Don't merge. Let me know first. I have (`delay` and `prevent_refresh`) placebos in there that should be removed first.~~ Ready.

### CODE REVIEW

File: doc/py3-cmd.rst [hunk 1]
```diff
@@ -18,16 +18,16 @@ To refresh all modules use the ``all`` keyword.
 
 .. code-block:: shell
 
-    # refresh any wifi modules
+    # refresh wifi modules
```
Comment (ultrabug) [inline doc/py3-cmd.rst]: module

File: py3status/command.py [hunk 2]
```diff
@@ -3,21 +3,73 @@
...
+    ('v', 'verbose', 'enable verbose mode'),
+]
+SUBPARSERS = [
+    ('click', 'click modules'),
+    ('refresh', 'refresh modules'),
+]
+CLICK_OPTIONS = [
+    ('button', 'specify a button number (default %(default)s)'),
+    ('index', 'specify an index value often found in modules'),
+    ('width', 'specify a width of the block, in pixel'),
```
Comment (ultrabug) [inline py3status/command.py]: sort them alphabetically

File: doc/py3-cmd.rst [hunk 3]
```diff
@@ -39,22 +39,27 @@ You can specify the button to simulate.
 
 .. code-block:: shell
 
-    # send a click event to the whatismyip module (button 1)
```
Comment (ultrabug) [inline doc/py3-cmd.rst]: will we still have that default button 1 behaviour?

File: doc/py3-cmd.rst (hunk 1 above)
Comment (lasers) [reply doc/py3-cmd.rst]: Plural. Add many as you want... It'll refresh all of them.

File: doc/py3-cmd.rst (hunk 3 above)
Comment (lasers) [reply doc/py3-cmd.rst]: Yes, sir. See `py3-cmd click`   ==> `--button INT (default 1)` (Not hardcoded).

File: py3status/command.py (hunk 2 above)
Comment (lasers) [reply py3status/command.py]: `width, height` belongs together (imho).
```
    ('button', 'specify a button number (default %(default)s)'),
    ('height', 'specify a height of the block, in pixel'),
    ('index', 'specify an index value often found in modules'),
    ('relative_x', 'specify relative X on the block, from the top left'),
    ('relative_y', 'specify relative Y on the block, from the top left'),
    ('width', 'specify a width of the block, in pixel'),
    ('x', 'specify absolute X on the bar, from the top left'),
    ('y', 'specify absolute Y on the bar, from the top left'),
```

File: doc/py3-cmd.rst (hunk 1 above)
Comment (ultrabug) [reply doc/py3-cmd.rst]: sure, but that's not what the example below shows, so in this case you only refresh the wifi module

File: doc/py3-cmd.rst (hunk 3 above)
Comment (ultrabug) [reply doc/py3-cmd.rst]: can we say it and show an example without --button then ?

File: py3status/command.py (hunk 2 above)
Comment (ultrabug) [reply py3status/command.py]: I know but I don't care they belong together, I'd just like them sorted plz

**Comment by ultrabug** [comment]
Awesome, thanks!

**Pull request merged**